buildscripts/ci/backend/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM library/ubuntu:20.04
+FROM library/ubuntu:22.04
```

Comment:
needed for master too
Sorry, I seem to have missed that in my earlier PR to change everything to Ubuntu 22.04

Reply:
Oops, this __is__ for master...